blocks/donation-form/class-give-donation-form-block.php

```diff
@@ -117,6 +119,18 @@ public function register_block() {
 					'id'                  => array(
 						'type' => 'number',
 					),
+					'className' => array(
+						'type' => 'string',
+						'default' => '',
+					),
+					'align' => array(
+						'type' => 'string',
+						'default' => '',
+					),
+					'extUtilities' => array(
```

Comment:
@mhimon Where I can find the `align` setting and the `extUtilities` setting because the `Advanced` tab only contains setting for custom classes.

Reply:
We can remove align attribute as that block doesn't have support for that. 

And extUtilities is in Advanced Inspector Controls.

![Screen Shot 2022-05-05 at 6 13 09 PM](https://user-images.githubusercontent.com/27549230/166920611-6d45cad8-f72b-41ab-9cab-58e7ad47f18c.png)

Reply:
@mhimon it seems that `Exntendify Utilities` is not the default setting field. Does it appear when you install plugin https://wordpress.org/plugins/extendify/

Reply:
Sorry! yes, I think so. Thanks man